--- im-ja-1.5/src/xim/Makefile.am.orig	2006-04-18 12:52:23.000000000 +0200
+++ im-ja-1.5/src/xim/Makefile.am	2024-07-20 17:23:09.608539902 +0200
@@ -87,5 +87,5 @@ im_ja_xim_server_CFLAGS =    \
     @DISABLE_ANTHY_FLAG@     \
     @DISABLE_SKK_FLAG@       \
     @X_CFLAGS@
-im_ja_xim_server_LDADD = IMdkit/libXimd.a
+im_ja_xim_server_LDADD = IMdkit/libXimd.a -lm
 im_ja_xim_server_LDFLAGS = @GTK_LIBS@ @GCONF_LIBS@ @X_LIBS@ @X_PRE_LIBS@ -lX11 @CANNA_LIBS@ @WNN_LIBS@ @ANTHY_LIBS@
